VadeCloud/manifest.json

```
@@ -25,5 +25,5 @@
   "name": "Vade Cloud",
   "slug": "vade-cloud",
   "uuid": "203d043e-a1ea-4b5c-81ef-6fc0f8a6a9d3",
-  "version": "1.0.1"
+  "version": "1.0.3"
```

Fix the version number.
```suggestion
  "version": "1.0.2"
```
Could you also add an entry in the CHANGELOG.md?
Thanks